platform/security/sts/security-sts-upbstvalidator/pom.xml

```diff
@@ -67,7 +80,7 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Export-Package/>
+                        <Bundle-Name>${project.name}</Bundle-Name>
```

Comment:
Why did we remove the Export-Package instruction here? This bundle shouldn't be exporting anything.